pipelines/smartseq2_single_sample/SS2SingleSample.wdl

```diff
@@ -0,0 +1,136 @@
+##### Naming Rules #####
+## ** Files should be named in CamelCase starting with a capital
+## ** Tasks should be named in CamelCase starting with a capital
+## ** Alias' should be named in CamelCase starting with a capital
+## ** Variable declarations should be lower_snake_case
+
+##### New Line Rules #####
+## ** New line before and after `calls`
+## ** New Line between task definitions
+## ** New lines between blocks in tasks
+## ** New line variable declarations that you want to group
+## ** New line between imports and workflow definition
+## ** No new lines at the start or end of any workflow/call/task definition block
+## ** No need for having two new lines anywhere
+## ** No new lines at the start of any blocks
+
+##### Spacing Rules #####
+## ** Space before `{` characters
+## ** Space after `,` characters
+## ** Space before and after operators e.g. `+`, `=`
+## ** Space after `:` character in runtime block
+## ** Space after `#` when creating a comment (or `##` if we decide thats our comment definer)
+
+##### Tab vs Space for Indents ######
+## ** Always use spaces? I think whats most frequently used is whenever you want to indent you use 2 spaces but we should decide what works best for us.
+
+##### Runtime Attributes ######
+## ** This will change quite a bit i believe when the new spec of WDL comes out (WDL SPEC 3) but we should define all values we care about
+## ** even if they are the default values.  e.g. write `cpu: 1` instead of leaving it out because its the default.
+
+##### Comment Rules ######
+## **  Should comments have one or two `##` before a comment.  Maybe two for workflow or task descriptions and one for any other kind of comment?
```

Comment:
One. Always. if the comment is defined by WDL as `#`, that's what we should use.

Reply:
I agree